pokemongo_bot/__init__.py

```diff
@@ -999,6 +1001,32 @@ def get_encryption_lib(self):
 
         return full_path
 
+    #I'm missing OSX libraries support, anyone that can help me on this.
+    def get_hash_lib(self):
+        if _platform == "Windows" or _platform == "win32":
+            # Check if we are on 32 or 64 bit
+            if sys.maxsize > 2**32:
+                file_name = 'niantichash64.dll'
+            else:
+                file_name = 'niantichash32.dll'
+        else:
+            file_name = 'libniantichash-linux-x86-64.so'
```

Comment:
No I don't think so, I haven't worked on mac ever so I don't know what to check the _platform variable against? care to help me on that?